api/models.py

```diff
@@ -15,7 +16,7 @@ class UserProfile(models.Model):
 	fullname = models.CharField(max_length=100, blank=False)
 	githubuser = models.CharField(max_length=100, null=False, blank=False)
 	email = models.EmailField(blank=False)
-	profile_picture = models.ImageField(blank = True)  
+	profile_picture = models.ImageField(upload_to=settings.MEDIA_UPLOAD_FOLDER, blank=True)  
```

Comment:
You're missing migration for this change. (just run ./manage.py makemigrations ) to create it.